index.html

```diff
@@ -0,0 +1,522 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: desktop version day#1</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <!-----<link rel="stylesheet" href="css/finish.css"--------->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+  <div class="wrapper">
+    <header class="header-2 background-color">            
+      <div class="logo-link">
+        <a href="./index.html" class="my-logo">My Logo</a>
+      </div>
+      <nav class="navbar-desk">
+        <ul class="menu-desk">
+          <li class="menu-bar-desk portfolio-link"> 
+            <a href="#portfolio"> Portfolio</a>
+          </li>
+          <li class="menu-bar-desk about-link"> 
+            <a href="#about"> About</a>
+          </li>
```

Comment:
- [ ] please remove this extra margin on the right.
![Screenshot (389)_LI](https://user-images.githubusercontent.com/65298048/187799966-6b397b1b-96ac-4758-8977-e33e19ea4ec1.jpg)